app/utils/cyaHelpers/item/ItemAnswersHelper.scala

```diff
@@ -157,4 +159,13 @@ class ItemAnswersHelper(
     args = itemIndex.display
   )
 
+  def packagesList: Seq[SummaryListRow] =
+    getAnswersAndBuildSectionRows(PackagesListSection(itemIndex))(packages)
+
+  def packages(packageIndex: Index): Option[SummaryListRow] = getAnswerAndBuildSectionRow[PackagesDomain](
```

Comment:
`package` instead of `packages`?

Reply:
For this method name, "package" is a protected name, hence we can't use it here except we wrap it in backticks. If you don't have any other concerns, I will suggest to leave it this way.